benchmark/process/next-tick-depth-args.js

```diff
@@ -2,13 +2,13 @@
 
 const common = require('../common.js');
 const bench = common.createBenchmark(main, {
-  millions: [12]
+  n: [12e6]
 });
 
 process.maxTickDepth = Infinity;
 
-function main({ millions }) {
-  var n = millions * 1e6;
+function main({ n }) {
+
 
```

Comment:
Please do not add double line breaks. This is a unnecessary change.

Reply:
changes made :)